web/themes/joinup/prototype/sass/components/_listing.sass

```diff
@@ -14,9 +14,12 @@
 .listing__item
   position: relative
   background: #fff
+  display: flex
+  align-items: stretch
 
 .listing__card
-  position: static
+  display: flex
```

Comment:
On the design content is centered - https://drive.google.com/file/d/0B31UGo3Wie5wZGRGMk9JWkN1UGM/view , and I think that it looks better than aligned to the top.